sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/analysis/json/SensitivityAnalysisParametersJsonModule.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
```

Comment:
Should the copyright be 2020? Or is it a moved file?

Reply:
It is a moved file (renamed but moved)